### PR TITLE
Adding the possibility of controlling the elements that will be treated

### PR DESCRIPTION
**Pull request by EmanuelAugusto**
In this pr I added the possibility for the implementer to control attributes that will be treated in addition to those already implemented in the package.

### CODE REVIEW

**Review by Pudge601 — DISMISSED** [review]
All looks good to me, thanks for the pull request!

File: src/Filter/RemoveAttributes.php [hunk 1]
```diff
@@ -17,7 +17,7 @@ class RemoveAttributes implements FilterInterface
 
     private AttributeFinder $attributeFinder;
 
-    public function __construct()
+    public function __construct($attributes = [])
```
Comment (chrisminett) [inline src/Filter/RemoveAttributes.php:20]: Can you add/update tests to cover this change, please?

File: src/Sanitizer.php [hunk 2]
```diff
@@ -16,8 +16,14 @@ class Sanitizer
      */
     private array $filters;
 
-    public function __construct()
+    private array $removeBlocks = ['script', 'iframe', 'object'];
+    
+    private array $removeAttributtes = [];
+
+    public function __construct(array $removeBlocks = [], array $removeAttributtes = [])
```
Comment (chrisminett) [inline src/Sanitizer.php:23]: I added type declarations here, but online editor added too many spaces :doh:. Dev dependency requirements have ECS included which you can run locally to fix this and the missing declaration in `RemoveAttributes`.

```sh
./vendor/bin/ecs --fix
```

Comment (chrisminett) [inline src/Sanitizer.php:23]: Can you add/update tests to cover this change, please?

File: src/Sanitizer.php [hunk 3]
```diff
@@ -16,8 +16,14 @@ class Sanitizer
...
     {
+        $this->removeBlocks = [...$this->removeBlocks, ...$removeBlocks];
+        $this->removeAttributtes = [...$this->removeAttributtes, ...$removeAttributtes];
```
Comment (chrisminett) [inline src/Sanitizer.php:26]: This could be simplified as there's no default on the class. The class property doesn't need an initial value.

**Review by chrisminett — CHANGES_REQUESTED** [review]
Thanks for the PR! I've added some minor tidy comments, but the main thing is to add/update unit tests to cover the new functionality.
I can have a look at that in the next few weeks, but please update if you're able. Thanks!

**Comment by chrisminett** [comment]
@EmanuelAugusto I've added tests in #10 and this has been released as [2.1.0](https://github.com/phlib/xss-sanitizer/releases/tag/2.1.0).
Thanks!

**Pull request merged**